dockerspawner/dockerspawner.py

```diff
@@ -147,6 +147,18 @@ def client(self):
         )
     )
 
+    internal_ip_network_name = Unicode(
+        "bridge",
+        config=True,
+        help=dedent(
+            """
+            Retrieve the internal IP from this docker network. Defaults to the default docker
```

Comment:
Can you make the mentioned update to the help string? "The name of the network..."